keps/sig-instrumentation/3466-kubernetes-component-health-slis/README.md

```diff
@@ -336,16 +336,7 @@ We will target this feature behind a flag `ComponentSLIs`
 
 ###### How can this feature be enabled / disabled in a live cluster?
 
-
-- [ ] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: `ComponentSLIs`
-  - Components depending on the feature gate:
-    + apiserver
-    + kubelet
-    + scheduler
-    + controller-manager
-    + kube-proxy
-
+Given this feature is purely in-memory, no enablement/disablement tests are needed.
```

Comment:
Sorry - maybe I wasn't clear.

This part (about the feature gate) should remain as it was.

The sentence you added should do to the question
```
###### Are there any tests for feature enablement/disablement?
```

Reply:
Done